src/hooks/useDitto.ts

```diff
@@ -23,12 +25,12 @@ export const useDitto = (props: useDittoProps) => {
       const frame = data[frameId];
       if (frame) {
         if (!blockId) {
-          return filterFrame(frame, filters);
+          return filterFrame(frame, variables, count, filters);
```

Comment:
```suggestion
          return filterFrame(frame, variables || {}, count, filters);
```
For reasons described [here](https://github.com/dittowords/ditto-react/pull/10/files#r784865890)

Reply:
👍